tests/core/test_snapshot_evaluator.py

```diff
@@ -158,7 +158,7 @@ def test_runtime_stages(capsys, mocker, adapter_mock, make_snapshot):
     @macro()
     def increment_stage_counter(evaluator) -> None:
         # Hack which allows us to intercept the different runtime stage values
-        print(f"RuntimeStage value: {evaluator.locals['runtime_stage'].value}")
+        print(f"RuntimeStage value: {evaluator.locals['runtime_stage']}")
```

Comment:
why does this call print?

Reply:
Because intercepting the macro call at runtime stages besides evaluation wasn't trivial for this test. The pattern we're using in the snapshot evaluator tests is to pass a `payload` dict in `evaluate` so it can populate it but that misses the load / create stages. See https://github.com/TobikoData/sqlmesh/blob/main/tests/core/test_snapshot_evaluator.py#L80.